tests/endtoend/test_eventhub_batch_functions.py

```diff
@@ -3,10 +3,16 @@
 import json
 import time
 import pathlib
+from datetime import datetime
+from dateutil import parser, tz
 
 from azure_functions_worker import testutils
 
 
+# This is EventHub (cardinality: many) E2E. Each testcase consists of 3 part:
+# 1. An eventhub_output_batch HTTP trigger for generating EventHub event
+# 2. An eventhub_multiple EventHub trigger for converting event into blob
+# 3. A get_eventhub_batch_triggered HTTP trigger for getting the event body
 class TestEventHubFunctions(testutils.WebHostTestCase):
```

Comment:
Ideally, you could use the [docstring comments](https://www.python.org/dev/peps/pep-0257/) for the comments where you explain methods and Classes. I am open to doing that in another PR.

```suggestion
class TestEventHubFunctions(testutils.WebHostTestCase):
    """Test EventHub Triggers and Bindings
    
    This is EventHub (cardinality: many) E2E. Each testcase consists of 3 part:
    1. An eventhub_output_batch HTTP trigger for generating EventHub event
    2. An eventhub_multiple EventHub trigger for converting event into blob
    3. A get_eventhub_batch_triggered HTTP trigger for getting the event body
    """
```